src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -443,18 +443,34 @@ private long GetMaxTombstonesEtagToDelete(TransactionOperationContext context, s
             {
                 var singleBackupStatus = _server.Cluster.Read(context, PeriodicBackupStatus.GenerateItemName(dbName, taskId));
                 if (singleBackupStatus == null)
+                {
+                    // no backup status for this task
                     continue;
+                }
 
-                if (singleBackupStatus.TryGet(nameof(PeriodicBackupStatus.LocalBackup), out BlittableJsonReaderObject localBackup) == false
-                    || singleBackupStatus.TryGet(nameof(PeriodicBackupStatus.LastRaftIndex), out BlittableJsonReaderObject lastRaftIndexBlittable) == false)
+                if (singleBackupStatus.TryGet(nameof(PeriodicBackupStatus.Error), out BlittableJsonReaderObject error) == false || error != null)
+                {
+                    // this backup task last status is error
                     continue;
+                }
+
+                if (singleBackupStatus.TryGet(nameof(PeriodicBackupStatus.LastFullBackupInternal), out DateTime? lastFullBackupInternal) == false || lastFullBackupInternal == null)
```

Comment:
I know that this is unchangeable and the moment, but why this is called 'Internal' ?